packages/shopify/package.json

```diff
@@ -46,8 +46,8 @@
     "vitest": "3.1.2"
   },
   "dependencies": {
-    "@coveo/headless": "3.23.1",
-    "@coveo/relay": "1.1.3",
+    "@coveo/headless": "3.24.0-pre.fc6c0cb3e8",
```

Comment:
:suspect:

Reply:
The feature relies on a recent update to `@coveo/headless`. Not referencing the alpha would mean that this change would have to span multiple weeks and releases? Once the current alpha version of headless is released I can update this to point to it

Reply:
Since its in the same mono repo, that's not how it works. 

This gets auto updated and kept in sync by the tooling of the repo.